js/src/personal/amanda/Landing.page.tsx

```diff
@@ -1,10 +1,14 @@
 import { Button, Center, Container, Title } from '@mantine/core'
 import { MouseEventHandler } from 'react'
 import styles from './Amanda.module.css'
+import { LightDarkButton } from '@/personal/amanda/LightDarkButton.tsx'
 
 export function LandingPage({ onClick }: { onClick: MouseEventHandler }) {
   return (
     <>
+      <Container className={styles.lightDark}>
+        <LightDarkButton />
+      </Container>
```

Comment:
nit: make a top "nav bar" that is shared across landing and Main page so the lightdarkbutton is shared and used once.